sources/RuleTab.cpp

```diff
@@ -38,10 +41,10 @@ RuleTab::RuleTab()
 
 	if (fRuleList->CountItems() == 0) {
 		BAlert* alert = new BAlert("Filer",
-			"It appears that there aren't any rules for "
+			B_TRANSLATE("It appears that there aren't any rules for "
 			"organizing files. Would you like Filer to "
-			"add some basic ones for you?",
-			"No", "Yes");
+			"add some basic ones for you?"),
+			B_TRANSLATE("No"), B_TRANSLATE("Yes"));
 
```

Comment:
Yes/No buttons are frowned upon. Buttons should be descriptive on their own, if possible. How about "No, thanks" and "Add default rules"?